fast64_internal/f3d/f3d_parser.py

```diff
@@ -901,10 +918,25 @@ def loadGeoFlags(self, command):
         mat.rdp_settings.g_shade = bitFlags & self.f3d.G_SHADE != 0
         mat.rdp_settings.g_cull_front = bitFlags & self.f3d.G_CULL_FRONT != 0
         mat.rdp_settings.g_cull_back = bitFlags & self.f3d.G_CULL_BACK != 0
+        if self.f3d.F3DEX_GBI_3:
+            mat.rdp_settings.g_attroffset_st_enable = bitFlags & self.f3d.G_ATTROFFSET_ST_ENABLE != 0
+            mat.rdp_settings.g_attroffset_z_enable = bitFlags & self.f3d.G_ATTROFFSET_Z_ENABLE != 0
+            mat.rdp_settings.g_packed_normals = bitFlags & self.f3d.G_PACKED_NORMALS != 0
+            mat.rdp_settings.g_lighttoalpha = bitFlags & self.f3d.G_LIGHTTOALPHA != 0
+            mat.rdp_settings.g_ambocclusion = bitFlags & self.f3d.G_AMBOCCLUSION != 0
+            mat.rdp_settings.g_fresnel = bitFlags & self.f3d.G_FRESNEL != 0
```

Comment:
if those values are only 0 or 1 would it be worth to use true/false? unless you plan on adding more possible values in the future

Reply:
The objects on the left are `bool`s (actually `BoolProperty`s), on the right it's checking if bits in an integer are set.